ingestfile

```diff
@@ -243,12 +246,18 @@ if
     echo "ERROR The MEDIA ID must only contain capital letters, numbers, hyphen and underscore"
     echo "Please reenter valid ID"
     MEDIAID=""
-    _ask_mediaid
+    if [ -z "${SILENT_RUNNING}" ] ; then
```

Comment:
This if and the one one line 256 might be unnecessary, these functions already check to see if the variable is unset before asking for it to be requested, else they don't do anything.